### DISCUSSION TITLE
Home Assistant Integration RC vs Final?

### DISCUSSION THREAD

**Discussion by ve6rah** [blakeblackshear/frigate#5981] [General]
Congrats on getting 0.12.0 to final! lots of hard work went into that I'm sure!

Now that 0.12.0 is the default frigate version, I note that it only works with HA integration 4.0.0, however I note that 4.0.0 is still only a release candidate and isn't final yet. Any news on the progress there so that the integration, and frigate are both matched to the latest version?

Thanks!

**Comment by NickM-27** [comment]
We're working on it, should be soon
